frontend/src/metabase/dashboard/containers/DashboardSharingEmbeddingModal.jsx

```diff
@@ -49,14 +46,15 @@ class DashboardSharingEmbeddingModal extends Component {
       isLinkEnabled,
       updateEnableEmbedding,
       updateEmbeddingParams,
+      onClose,
+      triggerElement,
       ...props
     } = this.props;
     if (!enabled) {
       return null;
     }
     return (
-      <ModalWithTrigger
-        ref={m => (this._modal = m)}
+      <Modal
```

Comment:
Converting this into a normal `Modal` since it's been converted into a controlled component now